admin/package.json

```diff
@@ -9,10 +9,10 @@
     "bower": "^1.3.12",
     "font-awesome": "^4.5.0",
     "font-awesome-webpack": "0.0.5-beta.2",
-    "json-loader": "^0.5.1",
     "jquery-datetimepicker": "^2.5.16",
+    "json-loader": "^0.5.1",
     "keen-js": "^3.0.0",
-    "knockout": "~3.2.0",
+    "knockout": "3.4.0",
```

Comment:
Why is this necessary?

Reply:
Somehow, Knockout@~3.2.0 does not seem to support CommonJS style `require('knockout')` imports. I believe https://github.com/knockout/knockout/pull/1526 is relevant. Bumping to 3.4.0 solves the problem.